README.rst

```diff
@@ -44,17 +44,17 @@ browser at:
 
 http://localhost:8080
 
-You may wish to make changes from the default configuration. This can be
-done in the ``config/galaxy.ini`` file. Tools can be either installed
-from the Tool Shed or added manually. For details please see the Galaxy
-wiki:
-
-https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/
+Configuration & Tools
+=====================
 
-Not all dependencies are included for the tools provided in the sample
-``tool_conf.xml``. A full list of external dependencies is available at:
+You may wish to make changes from the default configuration. This can be
+done in the ``config/galaxy.ini`` file.
 
-https://galaxyproject.org/admin/config/tool-dependencies/
+Tools can be either installed from the Tool Shed or added manually.
+ For details please see the `tutorial <https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/>`__.
+Note that not all dependencies for the tools provided in the
+``tool_conf.xml.sample`` are included. To install them please visit
+"Manage dependencies" in the admin interface.
```

Comment:
It's actually called "Manage Tool Dependencies", sorry for the previous misleading suggestion.

Reply:
True, it has been changed in `dev`, it's still "Manage Tool Dependencies" in 17.09. Not sure what is best here, but I suppose it's fine as it is now.

Reply:
Given our speed of updates in this readme we are fine looking a bit forward I think. :)